chia/consensus/blockchain.py

```diff
@@ -95,16 +93,12 @@ class Blockchain:
     # peak of the blockchain
     _peak_height: Optional[uint32]
     # All blocks in peak path are guaranteed to be included, can include orphan blocks
-    __block_records: dict[bytes32, BlockRecord]
+    _block_records: dict[bytes32, BlockRecord]
```

Comment:
There's no reason to use double-underscore here, and it does have a specific meaning that we don't actually want to invoke here. https://docs.python.org/3/tutorial/classes.html#private-variables